clients/admin-ui/src/features/user-management/NewUserForm.tsx

```diff
@@ -20,9 +23,11 @@ const NewUserForm = () => {
 
   const handleSubmit = (values: FormValues) => {
     const b64Password = utf8ToB64(values.password);
-    return createUser({ ...values, password: b64Password });
+    createUser({ ...values, password: b64Password });
+    router.push(`${USER_MANAGEMENT_ROUTE}`);
```

Comment:
I think we may still want the nice toasts that indicate the user was created successfully here. also I think we may want to redirect to the profile of this user? are we able to grab their created id from `createUser` and then build the URL off of that?

Reply:
Good call, I'll update to redirect to user profile, and add back the toasts for successful user creation